src/bicep/README.md

```diff
@@ -415,6 +415,27 @@ You can modify this naming convention to suit your needs. We recommend following
     }
     ```
 
+## Cleanup
+
+The Bicep/ARM deployment of Mission Landing Zone can be deleted with two steps:
+
+1. Delete all resource groups.
```

Comment:
I usually delete in bulk via the command line and let ARM figure it out. Will users deleting from the Portal have specific order of operations considerations?

Reply:
^ we can address this in another PR, of course. I assume it will take some investigation to work out.